File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_2/palindrome.java
public class palindrome {
    public static boolean palindromeNumber(int n){
        // Write your code here.
        int rev = 0;
        int temp = n;
        int digit =0;
        if(n<10){
            return true;
        }
        else{
        while(temp!=0){
            digit = temp%10;
            rev = 10*rev+digit;
            temp = temp/10;
        }
        return n==rev;
        }
    }
}

File: SecondYear/Junior-Hawk_ArchitAgarwal_2226cseai1186_2/Week_2/palindrome.cpp
bool palindrome(int n)
{
    int k = n;
    int s=0;
    while(k>0){
        s = s*10 + (k%10);
        k /= 10;
    }
    if(s==n){
        return true;
    }
    else{
        return false;
    }
}